src/hera/workflows/_runner/script_annotations_util.py

```diff
@@ -16,26 +15,22 @@
 from hera.shared._type_util import (
     get_unsubscripted_type,
     get_workflow_annotation,
-    is_subscripted,
     origin_type_issubtype,
     unwrap_annotation,
 )
 from hera.shared.serialization import serialize
 from hera.workflows import Artifact, Parameter
 from hera.workflows.artifact import ArtifactLoader
 from hera.workflows.io.v1 import (
-    Input as InputV1,
     Output as OutputV1,
```

Comment:
nit: would you be averse to removing the trailing comma so this reverts to a single import? If so, we can set `split-on-trailing-comma = False` [in the isort settings](https://docs.astral.sh/ruff/settings/#lint_isort_split-on-trailing-comma)

Reply:
I can fix these to be on one line, but I'd prefer not changing the setting – tried it and got this kind of diff:

```diff
--- a/examples/workflows/misc/template_level_volume.py
+++ b/examples/workflows/misc/template_level_volume.py
@@ -2,15 +2,7 @@
 
 See https://argo-workflows.readthedocs.io/en/latest/walk-through/volumes/"""
 
-from hera.workflows import (
-    Container,
-    ExistingVolume,
-    Parameter,
-    Resource,
-    Step,
-    Steps,
-    Workflow,
-)
+from hera.workflows import Container, ExistingVolume, Parameter, Resource, Step, Steps, Workflow
```

Which is uhhhh... not nice :)